source/speech/sayAll.py

```diff
@@ -104,8 +106,11 @@ def readText(
 		startPos: Optional[textInfos.TextInfo] = None,
 		nextLineFunc: Optional[Callable[[textInfos.TextInfo], textInfos.TextInfo]] = None,
 		shouldUpdateCaret: bool = True,
+		startedFromScript: bool = False,
```

Comment:
>It's None meaning that sayAll.SayAllHandler.startedFromScript should not be modified with this action. It should not be modified because this call is to resume an existing say all action, not to start it.

>Should I make it somewhat more explicit? With a comment or otherwise? - @CyrilleB79  https://github.com/nvaccess/nvda/pull/16826/files#r1674008395


Yes, please fix up the types and add a docstring for this function

```suggestion
		startedFromScript: bool | None = False,
```